internal/tiltfile/docker_compose.go

```diff
@@ -191,32 +192,42 @@ func parseDCConfig(ctx context.Context, configPath string) ([]dcService, error)
 	return services, nil
 }
 
-func (s dcService) ToManifest(dcConfigPath string) (manifest model.Manifest,
+func (s *tiltfileState) ToManifest(service dcService, dcConfigPath string) (manifest model.Manifest,
 	configFiles []string, err error) {
 	dcInfo := model.DockerComposeTarget{
 		ConfigPath: dcConfigPath,
-		YAMLRaw:    s.ServiceConfig,
-		DfRaw:      s.DfContents,
+		YAMLRaw:    service.ServiceConfig,
+		DfRaw:      service.DfContents,
 	}
+
+	dockerIgnores := pathsToDockerIgnores([]string{path.Dir(service.DfPath), path.Dir(dcConfigPath)})
```

Comment:
hmm, you should probs do all this AFTER we get mounts -- e.g. what if we're mounting `./foo/bar/baz`, which is its own git repo, but the dockerfile lives in like `./docker`? (idk how likely that case is... but trying to get gitignore/dockerignore for every mounted path as well as for dependencies is closest to how we do it for k8s)